renderer/Profiling.h

```diff
@@ -0,0 +1,51 @@
+/*****************************************************************************
+                    The Dark Mod GPL Source Code
+ 
+ This file is part of the The Dark Mod Source Code, originally based 
+ on the Doom 3 GPL Source Code as published in 2011.
+ 
+ The Dark Mod Source Code is free software: you can redistribute it 
+ and/or modify it under the terms of the GNU General Public License as 
+ published by the Free Software Foundation, either version 3 of the License, 
+ or (at your option) any later version. For details, see LICENSE.TXT.
+ 
+ Project: The Dark Mod (http://www.thedarkmod.com/)
+ 
+******************************************************************************/
+
+#ifndef __PROFILING_H__
+#define __PROFILING_H__
+
+#include "qgl.h"
+#include "tr_local.h"
+
+extern idCVar r_useDebugGroups;
+extern idCVar r_glProfiling;
+
+void EnterProfilingSection( const char* section );
```

Comment:
I'll probably grumble about several things here.

The first one is naming. I know that renderer is mostly C-style code, with separate functions declared in h-files and defined somewhere in cpp. But anyway it's better to start all header-available function names with common prefix. If it is "Profiling", then better rename functions to `ProfilingLeaveSection` and `ProfilingDisplayInfo`.
And when you change all names to have similar prefix, it will become apparent that it makes more sense to group the functions into a namespace at least (or even class) =) Of course, if we follow the C-style, then the only grouping that we can use is having a common prefix in function names =)

The second one is about exposing `EnterProfilingSection` and `LeaveProfilingSection`. What is the intended usage of it? Right now `GlProfileScope` is guarded against absence of GL functionality (probably not needed) and also checks for cvars. The functions in question do not check for anything.